content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -6,7 +6,7 @@
   width: $link-menu-button-size;
   height: $link-menu-button-size;
   background-color: $white;
-  background-image: url('img/glyph-more-16.svg');
+  background-image: url($image-path + 'glyph-more-16.svg');
```

Comment:
Unrelated annoying comment, but I was playing with [**sass-inline-svg**](http://npm.im/sass-inline-svg) a couple weeks ago and we could easily inline all the SVGs into the CSS which may reduce loads/lag.

May be as simple as specifying the SVG inliner in an exported function and then specifying the function file using the `--functions` flag when calling node-sass:

```js
// funcs.js
const path = require("path");

const inliner = require('sass-inline-svg');

const SVG_PATH = path.join(__dirname, 'static', 'images');

module.exports = {
  svg: inliner(SVG_PATH)
};
```

```sh
$ node-sass content-src/main.scss -o data/content --functions ./funcs.js
```

... Where we change these `url()` calls to our custom `svg()` function:

```sass
.logo-icon {
  background: svg("glyph-bookmark-16-blue.svg"); // `$image-path` is defined in ./funcs.js
  height: 20px;
  width: 20px;
}
```